railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+ruby <%= "'#{RUBY_VERSION}'" %>
```

Comment:
@schneems - I can't see how that would happen, as long as the `.ruby_version` is committed. Am I missing something?